cli/package.json

```diff
@@ -117,9 +120,20 @@
       "import": "./index.mjs",
       "require": "./index.js"
     },
+    "./vue": {
+      "import": "./vue/dist/cypress-vue.esm-bundler.js",
+      "require": "./vue/dist/cypress-vue.cjs.js"
+    },
     "./package.json": {
       "import": "./package.json",
       "require": "./package.json"
+    },
+    "./react": {
+      "import": "./react/dist/cypress-react.esm-bundler.js",
+      "require": "./react/dist/cypress-react.cjs.js"
+    },
+    "./mount-utils": {
+      "require": "./mount-utils/dist/index.js"
```

Comment:
No import entry for mount-utils?

Reply:
No it wasn't built for ESM.

Because only cypress-react and cypress-vue consume it and **they** have an esm export it's not a problem.

We have tech debt to convert all of our npm/* packages to be esm-first and share the same rollup + typescript compilation settings.